src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -399,6 +408,52 @@ export const VolunteerFormCalendar = ({
   );
 };
 
+const fileFilters = (
+  fileRegex: any,
+  calendarData: {
+    id: number;
+    content: { files: any[] };
+  }
+) =>
+  JSON.stringify(
+    calendarData?.content?.files
+      .map(({ file_name: infoText, mime_type: mimeType, url: uri, id: fileId }: any) => {
+        const isFile = fileRegex.test(infoText);
+
+        if (isFile) return { entryId: calendarData?.id, fileId, infoText, mimeType, uri };
+        else return;
+      })
```

Comment:
this looks heavy on first read. i will refactor this to multiple lines for readability

Reply:
01b627a